Added default GitHub URL value to omnibus default attributes (#2360)
chef/supermarket: docs-chef-io/content/supermarket/config_rb_supermarket.md

: The application client secret that is used to authenticate Supermarket to GitHub. Default value: `nil`.

`default['supermarket']['github_url']`

> : The URL of the GitHub that Supermarket connects to. Default value:`https://github.com/`.

Reviewer: ```suggestion
: The GitHub URL that Supermarket connects to. Default value:`https://github.com/`.
```

### GitHub Enterprise

Use these settings to integrate Supermarket with GitHub Enterprise.

`default['supermarket']['github_access_token']`

: The access token created from your GitHub Enterprise account. Default value:`nil`.

`default['supermarket']['github_key']`

: The application client ID that is used to authenticate Supermarket to GitHub Enterprise. Default value: `nil`.

`default['supermarket']['github_secret']`

: The application client secret that is used to authenticate Supermarket to GitHub Enterprise. Default value: `nil`.

`default['supermarket']['github_enterprise_url']`

> : The URL of the GitHub Enterprise that Supermarket connects to. Default value:`nil`.

Reviewer: ```suggestion
: The GitHub Enterprise URL that Supermarket connects to. Default value:`nil`.
```

`default['supermarket']['github_client_option_authorize_url']`

> : The URL of the GitHub Enterprise Authorize For Example `https://github.YOURDOMAIN.com/login/oauth/authorize`. Default value:`nil`.

Reviewer: ```suggestion
: The URL used to authorize with your implementation of GitHub Enterprise Server. For example: `https://github.example.com/login/oauth/authorize`. Default value: `nil`.
```

`default['supermarket']['github_client_option_site']`

> : The URL of the GitHub Enterprise Site For Example `https://github.YOURDOMAIN.com/api/v3`. Default value:`nil`.

Reviewer: ```suggestion
: The URL for your implementation of GitHub Enterprise. For example: `https://github.example.com/api/v3`. Default value: `nil`.
```

`default['supermarket']['github_client_option_access_token_url']`

> : The URL of the GitHub Enterprise Access Token For Example `https://github.YOURDOMAIN.com/login/oauth/access_token`. Default value:`nil`.

Reviewer: ```suggestion
: The URL of the GitHub Enterprise access token: For example: `https://github.example.com/login/oauth/access_token`. Default value:`nil`.
```
